package.json

```diff
@@ -26,28 +26,30 @@
   },
   "dependencies": {
     "gulp-eslint-if-fixed": "^1.0.0",
+    "natives": "^1.1.6",
```

Comment:
ideally we want to use hard fixed versions. (without the `^` in these cases as it does affect the lock files)

Reply:
I see. I used "--save-dev". It automatically adds ^. How to avoid npm adding it? Or using yarn doesn't add ^?